app/forms/security_event_form.rb

```diff
@@ -148,24 +151,32 @@ def record_already_exists?
   end
 
   def validate_iss
-    errors.add(:iss, 'invalid issuer') if service_provider.blank?
+    errors.add(:iss, 'invalid issuer', type: :invalid_issuer) if service_provider.blank?
   end
 
   def validate_aud
     return if jwt_payload.blank?
     return if jwt_payload['aud'] == api_risc_security_events_url
 
-    errors.add(:aud, t('risc.security_event.errors.aud_invalid', url: api_risc_security_events_url))
+    errors.add(
+      :aud,
+      t('risc.security_event.errors.aud_invalid', url: api_risc_security_events_url),
+      type: :aud_invalid,
+    )
     @error_code = ErrorCodes::JWT_AUD
   end
 
   def validate_event_type
     if event_type.blank?
-      errors.add(:event_type, t('risc.security_event.errors.event_type_missing'))
+      errors.add(
+        :event_type, t('risc.security_event.errors.event_type_missing'),
+        type: :event_type_missing
+      )
     elsif !SecurityEvent::EVENT_TYPES.include?(event_type)
       errors.add(
         :event_type,
-        t('risc.security_event.errors.event_type_unsupported', event_type: event_type),
+        t('risc.security_event.errors.event_type_unsupported', event_type: :"#{event_type}"),
```

Comment:
I don't think we need to convert `event_type` to a symbol like this, and we could just do `.to_sym` if we did need to

```suggestion
        t('risc.security_event.errors.event_type_unsupported', event_type: event_type),
```